plugins/woocommerce/includes/rest-api/Controllers/Version3/class-wc-rest-product-variations-controller.php

```diff
@@ -614,7 +614,9 @@ public function get_item_schema() {
 				'status'                => array(
 					'description' => __( 'Variation status.', 'woocommerce' ),
 					'type'        => 'string',
-					'default'     => ProductStatus::PUBLISH,
+					// Not using ProductStatus constants here due the class not being loaded upon installation.
+					// https://github.com/woocommerce/woocommerce/issues/37464
```

Comment:
Sorry, I suggested this change but it seems PHPCS doesn't like it.

```suggestion
					// See: https://github.com/woocommerce/woocommerce/issues/37464.
```

(I'm not using `@see` or `@link` because those are really only valid in an actual `/**` docblock, though that's another possibility.)